samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -268,6 +301,46 @@ def _setup_log_file(log_file):
 
         return open(log_file, 'wb')
 
+    @staticmethod
+    def _get_debug_context(debug_port, debug_args, debugger_path):
+        """
+        Creates a DebugContext if the InvokeContext is in a debugging mode
+
+        Parameters
+        ----------
+        debug_port int
+             Port to bind the debugger to
+        debug_args str
+            Additional arguments passed to the debugger
+        debugger_path str
+            Path to the directory of the debugger to mount on Docker
+
+        Returns
+        -------
+        samcli.commands.local.lib.debug_context.DebugContext
+            Object representing the DebugContext
+
+        Raises
+        ------
+        samcli.commands.local.cli_common.user_exceptions.DebugContext
+            When the debugger_path is not valid
+        """
+        if debug_port and debugger_path:
+            try:
+                debugger = Path(debugger_path).resolve(strict=True)
+            except OSError as error:
+                if error.errno == errno.ENOENT:
+                    raise DebugContextException("'debugger_path' could not be found.")
```

Comment:
Minor comment: Instead of the string 'debugger_path', can you add the actual value of variable debugger_path  to the message? like '/usr/loca/ is not found'